src/Autohide.js

```diff
@@ -104,6 +91,36 @@ define(function (require, exports, module) {
         }
     }
 
+    /**
+     * Insert a placeholder on right.
+     */
+    function showPlaceholder() {
+        var toolbarPx = $("#main-toolbar:visible").width() || 0;
+        $mainView.append($placeholder);
+        $placeholder.css("width", "20px");
+        $placeholder.css("right", toolbarPx + "px");
+        $content.css("right", ($placeholder.width() || 0) + toolbarPx + "px");
+        $content.bind("transitionend.outline", function () {
+            addSidebarPlusTransition();
+        });
+        $placeholder.hover(function () {
+            var timer = $(this).data("hover");
```

Comment:
Please rename `"hover"` to something like `"expose-delay-timer"` (feel free to find something shorter).